### PR TITLE
Colorable decorative wings

### PR DESCRIPTION
**Pull request by Linzolle**
## About The Pull Request
NOTE: This PR assumes that https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9520 is merged.
Lets you colour those wings, as well as some fixes for them.

## Why It's Good For The Game
More customization, purely cosmetic.

## Changelog
:cl:
add: Decorative and insect wings can now be coloured individually, similar to horns.
fix: people getting assigned wings if their savefile is old.
del: wings that take the hair colour. Unnecessary if you can just set the colour.
tweak: Cannot make horns pure black. Trying to do so will set it to a default value.
/:cl:

### CODE REVIEW

File: code/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 1]
```diff
@@ -223,3 +215,18 @@
 /datum/sprite_accessory/insect_wings/feathery
 	name = "Feathery"
 	icon_state = "feathery"
+
+/datum/sprite_accessory/insect_fluff/colored1
```
Comment (Poojawa) [inline code/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/insect_wings/colored1
```

File: code/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 2]
```diff
@@ -223,3 +215,18 @@
...
+	name = "Colored (Primary)"
+	icon_state = "snowplain"
+	color_src = MUTCOLORS
+
+/datum/sprite_accessory/insect_fluff/colored2
```
Comment (Poojawa) [inline code/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/insect_wings/colored2
```

File: code/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 3]
```diff
@@ -223,3 +215,18 @@
...
+	name = "Colored (Secondary)"
+	icon_state = "snowplain"
+	color_src = MUTCOLORS2
+
+/datum/sprite_accessory/insect_fluff/colored3
```
Comment (Poojawa) [inline code/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/insect_wings/colored3
```

File: code/modules/mob/dead/new_player/sprite_accessories/wings.dm (hunk 1 above)
Comment (Linzolle) [reply code/modules/mob/dead/new_player/sprite_accessories/wings.dm]: these are actually not wing sprites, I have no idea why they're in the wings sprite accessory file. they are in fact the insect neck fluff

Comment (Linzolle) [reply code/modules/mob/dead/new_player/sprite_accessories/wings.dm]: just removed them because they're unnecessary anyway

**Review by nik707 — DISMISSED** [review]

**Review by nik707 — APPROVED** [review]
shut up bot

**Pull request merged**